test/e2e_node/image_volume.go

```diff
@@ -146,29 +166,123 @@ var _ = SIGDescribe("ImageVolume", nodefeature.ImageVolume, func() {
+		var selinuxOptions *v1.SELinuxOptions
+		if selinux.GetEnabled() {
+			selinuxOptions = &v1.SELinuxOptions{
+				User:  defaultSELinuxUser,
+				Role:  defaultSELinuxRole,
+				Type:  defaultSELinuxType,
+				Level: defaultSELinuxLevel,
+			}
+			ginkgo.By(fmt.Sprintf("Using SELinux on pod: %v", selinuxOptions))
+		}
+
 		createPod(ctx,
+			podName,
+			"",
 			[]v1.Volume{{Name: volumeName, VolumeSource: v1.VolumeSource{Image: &v1.ImageVolumeSource{Reference: invalidImageRef}}}},
 			[]v1.VolumeMount{{Name: volumeName, MountPath: volumePathPrefix}},
+			selinuxOptions,
 		)
 
-		ginkgo.By(fmt.Sprintf("Waiting for the pod (%v/%v) to fail", f.Namespace.Name, podName))
+		ginkgo.By(fmt.Sprintf("Waiting for the pod (%s/%s) to fail", f.Namespace.Name, podName))
 		err := e2epod.WaitForPodContainerToFail(ctx, f.ClientSet, f.Namespace.Name, podName, 0, images.ErrImagePullBackOff.Error(), time.Minute)
-		framework.ExpectNoError(err, "Failed to await for the pod to be running: (%v/%v)", f.Namespace.Name, podName)
+		framework.ExpectNoError(err, "Failed to await for the pod to be running: (%s/%s)", f.Namespace.Name, podName)
 	})
 
 	f.It("should succeed if image volume is not existing but unused", func(ctx context.Context) {
+		var selinuxOptions *v1.SELinuxOptions
+		if selinux.GetEnabled() {
+			selinuxOptions = &v1.SELinuxOptions{
+				User:  defaultSELinuxUser,
+				Role:  defaultSELinuxRole,
+				Type:  defaultSELinuxType,
+				Level: defaultSELinuxLevel,
+			}
+			ginkgo.By(fmt.Sprintf("Using SELinux on pod: %v", selinuxOptions))
+		}
+
 		createPod(ctx,
+			podName,
+			"",
 			[]v1.Volume{{Name: volumeName, VolumeSource: v1.VolumeSource{Image: &v1.ImageVolumeSource{Reference: invalidImageRef}}}},
 			nil,
+			selinuxOptions,
 		)
 
-		ginkgo.By(fmt.Sprintf("Waiting for the pod (%v/%v) to be running", f.Namespace.Name, podName))
+		ginkgo.By(fmt.Sprintf("Waiting for the pod (%s/%s) to be running", f.Namespace.Name, podName))
 		err := e2epod.WaitForPodNameRunningInNamespace(ctx, f.ClientSet, podName, f.Namespace.Name)
-		framework.ExpectNoError(err, "Failed to await for the pod to be running: (%v/%v)", f.Namespace.Name, podName)
+		framework.ExpectNoError(err, "Failed to await for the pod to be running: (%s/%s)", f.Namespace.Name, podName)
 
 		ginkgo.By(fmt.Sprintf("Verifying the volume mount is not used for path: %s", volumePathPrefix))
 
 		output := e2epod.ExecCommandInContainer(f, podName, containerName, "/bin/ls", filepath.Dir(volumePathPrefix))
 		gomega.Expect(output).NotTo(gomega.ContainSubstring(strings.TrimPrefix(volumePathPrefix, "/")))
 	})
+
+	f.It("should succeed with multiple pods and same image on the same node", func(ctx context.Context) {
+		node, err := e2enode.GetRandomReadySchedulableNode(ctx, f.ClientSet)
+		framework.ExpectNoError(err, "Failed to get a ready schedulable node")
+
+		baseName := "test-pod"
+		anotherSELinuxLevel := "s1:c6,c10"
```

Comment:
After re-running the new test, the job still fails with the same error.

Reply:
Can we try `s0`? Like 
```suggestion
		anotherSELinuxLevel := "s0:c100,c200"
```

Reply:
okay, updated. let's see the coming result.

Reply:
That worked :+1:

Reply:
@saschagrunert 
`s1` is a valid value for SELinux label according to https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/using_selinux/using-multi-level-security-mls_using-selinux#using-multi-level-security-mls_using-selinux. 

Is it a known limitation in k/k or `cri` implementation? I have no environment with SELinux enabled to test it.

Reply:
I assume the limitation is the CI. I'm not exactly sure if we can utilize `s1` or not.

Reply:
I create a draft PR to test it in https://github.com/kubernetes/kubernetes/pull/127237

Reply:
I can not find an example using s1-15 in the website and kubernetes repo. all examples are using s0. Let's ignore this test for now. 

If someone meets this issue, we can investigate it later.

What do you think? If you agree it, can you give me a LGTM on this PR? @saschagrunert @SergeyKanzhelev

Reply:
https://github.com/kubernetes/kubernetes/pull/127237 also fails for the same reason when creating a pod with the s1 label even if the pod does not use an image as volume.